Directory.Build.props

```diff
@@ -84,7 +84,8 @@
     <RuntimeInstallerBaseName>aspnetcore-runtime</RuntimeInstallerBaseName>
     <TargetingPackInstallerBaseName>aspnetcore-targeting-pack</TargetingPackInstallerBaseName>
 
-    <!-- Produce targeting pack installers/packages once per major.minor except in extraordinary cases i.e. 3.1.10. -->
+    <!-- Produce targeting pack installers/packages once per major.minor except in extraordinary cases i.e. 3.1.13 win-arm64. -->
+    <IsTargetingPackBuilding Condition=" '$(AspNetCorePatchVersion)' == '13' AND '$(TargetArchitecture)' == 'arm64' AND $([MSBuild]::IsOSPlatform('Windows')) ">true</IsTargetingPackBuilding>
```

Comment:
Are we sure about 13 being the version of the special build?

Reply:
Should be 14

Reply:
My bad, I need to rebase on your branding update.